docs/docsite/rst/intro_windows.rst

```diff
@@ -312,37 +312,46 @@ What modules are available
 Most of the Ansible modules in core Ansible are written for a combination of Linux/Unix machines and arbitrary web services, though there are various
 Windows-only modules. These are listed in the `"windows" subcategory of the Ansible module index <http://docs.ansible.com/list_of_windows_modules.html>`_.
 
-In addition, the following core modules work with Windows::
-
-    fetch
-    raw
-    script
-    slurp
-    template
-    add_host
-    assert
-    debug
-    fail
-    group_by
-    include_vars
-    meta
-    pause
-    set_fact
-
+In addition, the following core modules/action-plugins work with Windows:
+
+* add_host
+* assemble
+* assert
+* async
+* debug
+* fail
+* fetch
+* group_by
+* include_vars
+* meta
+* pause
+* raw
+* script
+* set_fact
+* setup
+* slurp
+* template (also: win_template)
 
 Some modules can be utilised in playbooks that target windows by delegating to localhost, depending on what you are
-attempting to achieve.  For example, assemble can be used to create a file on your ansible controller that is then 
-sent to your windows targets using win_copy.
+attempting to achieve.  For example, `assemble` can be used to create a file on your ansible controller that is then 
+sent to your windows targets using `win_copy`.
 
-In many cases, it may not be necessary to even write or use an Ansible module.
+In many cases, there isn't even a need to use or write an Ansible module.
```

Comment:
How about 'In many cases, there is no need to use or write and Ansible module.'  I like that you avoided the split infinitive but now the 'even' just seems unnecessary.  This is super-picky of me - keep it as is if you prefer.